src/Php72/Php72.php

```diff
@@ -80,4 +83,31 @@ public static function php_os_family()
 
         return isset($map[PHP_OS]) ? $map[PHP_OS] : 'Unknown';
     }
+
+    public static function spl_object_id($object)
+    {
+        if (null === self::$hashMask) {
```

Comment:
not sure it would help, L108 and L111 are not the same, so that would require duplicating L111, is that what you mean?